docker/Dockerfile.kubectl

```diff
@@ -1,5 +1,8 @@
 FROM istionightly/base_debug
 # Image for post install jobs
 
-# This container should only contain kubectl
-ADD kubectl /usr/bin
+ARG K8S_VER=v1.10.4
+
+# Include only kubectl in the image
+RUN curl -kLo /tmp/kubectl https://storage.googleapis.com/kubernetes-release/release/${K8S_VER}/bin/linux/amd64/kubectl && \
```

Comment:
in past reviews, @costinm was pretty strongly in favor of the current implementation vs using curl to install kubectl.  I'd recommend modifying the script that downloads kubectl to download both amd64 and darwin versions - postfix them with an architecture type and then handle the installation as a runtime argument to this dockerfile.